### DISCUSSION TITLE
Analytics did not track screen name

### DISCUSSION THREAD

**Discussion by teriologia** [invertase/react-native-firebase#6431] [Q&A]
Hi I've  issue about screen tracking. when I'm using react-navigation I want to get screen name but I cannot see and info about which screen displaying.

this is my code.
![image](https://user-images.githubusercontent.com/35257086/181768471-04e597ac-92ee-4e3c-9945-ae1dca636686.png)

this is log. I want to see screen name and class 
![image](https://user-images.githubusercontent.com/35257086/181768399-66065b36-6faf-4bf1-9442-dd34ac0052bc.png)

how can I solve this problem can someone help me pls thanks :)

**Comment by mikehardy** [comment]
I dunno - you might like this duplicate / similar issue though https://github.com/invertase/react-native-firebase/issues/6266 - it has the code I personally use

**Reply by gonzalo-palummo** [reply]
That's not the same, I have the same problem that @teriologia has. I'm already tracking screens the right way, but on User Explorer feature it's only showing the event name (screen_view) instead of show also the screen name

**Comment by gonzalo-palummo** [comment]
The same to me. I'm already tracking screens the right way, but on User Explorer feature it's only showing the event name (screen_view) instead of show also the screen name. On Firebase dashboard I can see screen names being tracked correctly
